pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -459,7 +459,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 						Expr:  intstr.FromString("kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes < 52428800 or kubevirt_vm_container_free_memory_bytes_based_on_rss < 52428800"),
 						For:   "1m",
 						Annotations: map[string]string{
-							"description": "Container {{ $labels.container }} in pod {{ $labels.pod }} in namespace {{ $labels.namespace }} free memory is less than 20 MB and it is close to requested memory",
+							"description": "Container {{ $labels.container }} in pod {{ $labels.pod }} in namespace {{ $labels.namespace }} free memory is less than 50 MB and it is close to requested memory",
```

Comment:
Would it be possible to move the actual limit into a constant somewhere and generate this string dynamically to avoid changes like this in the future?

Reply:
@lyarwood we would need to add some external dependency (like https://pkg.go.dev/github.com/dustin/go-humanize) or a big chunk of code to do the conversions between `52428800` and `50 MB`, is it worth it?

Reply:
`k8s.io/apimachinery/pkg/api/resource` could do this for us if we really need to use a different unit, or we just list the bytes value from the rule itself.  Apologies for making this overly complicated, just trying to avoid issues like this again.

Reply:
@lyarwood Per what I see, it works but we can only output the string format as "50Mi", looks okay for me, wdyt?

Reply:
Yeah that would work for me, thanks!